static/js/tools/timeline/chart_region.tsx

```diff
@@ -50,10 +55,12 @@ class ChartRegion extends Component<ChartRegionPropsType> {
   bulkDownloadLink: HTMLAnchorElement;
   bqLink: HTMLAnchorElement;
   allStatData: { [key: string]: StatData };
+  metadataMap: Record<string, Record<string, StatMetadata>>;
```

Comment:
nit: mind adding a comment describing the keys and values of the map?

Reply:
done